docs/docs/Configuration/configuration-authentication.md

```diff
@@ -24,6 +24,11 @@ You can copy the contents of this file into your own `.env` file and replace the
 
 ### LANGFLOW_AUTO_LOGIN
 
+:::important
+As of Langflow v1.5, all API requests require a Langflow API key, even when `AUTO_LOGIN` is enabled.
+The MCP-related endpoints will continue to require no authentication when `AUTO_LOGIN=true`.
+:::
+
```

Comment:
Please align this content with any changes you also make here: https://github.com/langflow-ai/langflow/pull/8721/files#diff-7f2ed0bc939770d71aa9d769af6e37d4432766134e78c9f8968b0342dcc8a54cR12